Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -246,13 +246,16 @@ function Run-Command
            cmd.exe /c "$command"
         }
 
+        if ($LastExitCode -ne 0)
+        {
+            throw [System.Management.Automation.RuntimeException] "$command failed with exit code $LastExitCode"
+        }
```

Comment:
Already there is try catch still you checked error on basis of $LastExitCode , I assume that you have done this because even your command fails during execution your task is not failing. This can be done by redirecting error stream in that way you will get actual error being thrown by exe. 
Ref. - https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/AzureRmWebAppDeployment/Utility.ps1#L213